.circleci/config.yml

```diff
@@ -123,21 +123,21 @@ workflows:
           context:
             - slack
     triggers:
-      - schedule: 
+      - schedule:
           cron: "0 * * * *"
           filters:
             branches:
               only:
                 - main
-  main:
+  pr-checks:
+    unless:
+        equal: [ main, << pipeline.git.branch >> ]
     jobs:
       - golang-lint
       - golang-modules-tidy
-      - golang-test:
-          context:
-            - slack
+      - golang-test
       - check-codegen
       - check-forge-fmt
-      - check-security-configs:
-          context:
-            - slack
+      - check-security-configs
```

Comment:
The hourly job only runs the `golang-test` and `check-security-configs` jobs. Just want to confirm those are the only ones we need on the schedule for now? It sounds correct based on the job names, but figured I'd double check.

Reply:
Yeah, these are the only ones whose pass/fail nature can be changed without altering the source code.